### ISSUE TITLE
Idea to compare websites for the same keyword

### ISSUE THREAD

**Issue by lionep** [towfiqi/serpbear#5]
Here is a proposal of feature (disclaimer, I'm not used to any SERP tool, so I don't know if it's something standard or not) : 

The user can add the same keyword for many websites (its own, and some competitors websites), and in a page related to this keyword, he can see the ranking of each website.

For example : 
on the keyword "A", the website 1 has gained 2 places (ranked 12), the website 2 has lost one (ranked 3), the website 3 remains in same position (ranked 6)...

Let me know if you think it's meaningless ;)

**Comment by towfiqi** [comment]
Its not meaningless at all. Some of the newer more robust SERP trackers actually have this feature.

Its possible, and I will think about it. Thanks for the suggestion :)

**Comment by s3w47m88** [comment]
I agree with @lionep . This is one of the most valuable features of SEMRush for my purposes. Tracking how our clients perform against competitors. I would be happy to have my staff contribute this @towfiqi .
